keras_retinanet/models/retinanet.py

```diff
@@ -57,14 +57,14 @@ def default_classification_model(
             filters=classification_feature_size,
             activation='relu',
             name='pyramid_classification_{}'.format(i),
-            kernel_initializer=keras.initializers.normal(mean=0.0, stddev=0.01, seed=None),
+            kernel_initializer=keras.initializers.he_normal(seed=None),
```

Comment:
Why `he_normal`? Would it be possible to use [`random_normal_initializer`](https://www.tensorflow.org/api_docs/python/tf/random_normal_initializer) instead? Would that be backwards compatible?

Reply:
normal and random_normal was not existent in the newest version, but we can use the RandomNormal class directly I think. I just liked how he_normal determines the stdev automatically, but this should indeed be another PR